Makefile

```diff
@@ -447,7 +447,7 @@ cpplint:
 	--exclude_path src/operator/contrib/ctc_include
 
 pylint:
-	pylint python/mxnet tools/caffe_converter/*.py --rcfile=$(ROOTDIR)/tests/ci_build/pylintrc
+	pylint --rcfile=$(ROOTDIR)/tests/ci_build/pylintrc --ignore-patterns=".*\.so$$,.*\.dll$$,.*\.dylib$$" python/mxnet tools/caffe_converter/*.py
```

Comment:
?

Reply:
You get an error when there's an mxnet binary in python/mxnet which is one of the paths which is searched for libraries, so sometimes there's an mxnet lib there.